Integrations/integration-BitDam.yml

```diff
@@ -111,10 +117,12 @@ script:
 
 
     def verdict_to_dbotscore(verdict):
-        if VERDICT_CLEAN == verdict:
+        if VERDICT_APPROVED == verdict:
```

Comment:
You tell me. I didn't address the  error, benign and timeout verdicts at all in the dbotscore because I didn't want to make these calls myself.  are timeout and error considered 'suspicious'?  are they considered 'unknown'? it seems like product decisions. We can talk of course :-)

Reply:
If benign means the file is clean, it is the same as approved from what I understand.
Other verdicts that are neither benign nor malicious should be DBOTCOKE UNKNOWN. (unless you have suspicious, but that's not the case from what I understand.)

Reply:
benign means BitDam's doesn't support this file type so it will not be scanned. It might be clean and it might be malicious but BitDam won't scan it.